src/chapters/configuring.xml

```diff
@@ -3229,11 +3248,73 @@ color9: rgb:80/00/80</programlisting>
                                     <para><indexterm>
                                             <primary>SSH</primary>
                                             <secondary>command</secondary>
-                                        </indexterm>The command to execute over the SSH session, if
-                                        any. This parameter is optional. If not specified, the SSH
+                                        </indexterm>By default, SSH sessions will
+                                        start an interactive shell.  The shell
+                                        that is used will be determined by the
+                                        SSH server, normally by reading the user's
+                                        default login shell specified in
+                                        <filename>/etc/passwd</filename>.  If you
+                                        wish to override this and run a specific
+                                        command, you can do so by filling in a
+                                        value for this parameter.  Assuming the
+                                        server allows arbitrary commands to be
+                                        run, the session will attempt to start
+                                        the specified command in place of the
+                                        shell.  This parameter is optional. If not specified, the SSH
                                         session will use the user's default shell.</para>
                                 </entry>
                             </row>
+                            <row>
+                                <entry><parameter>timezone</parameter></entry>
```

Comment:
If unsure about repurposing the "running a command" section, this could be documented within the "locale" section added recently via #117.

Reply:
Yes, this makes complete sense.